src/Driver.h

```diff
@@ -69,6 +69,17 @@ class Driver : public CBase_Driver<Data> {
     cb.send();
   }
 
+  void broadcastDecomposition(const CkCallback& cb) {
```

Comment:
Since this is not user-facing, and provides a clean way for users to PUP their decompositions I thought this was acceptable. I wanted to use a Charm++ message but I looked through the docs and I could not find a way for a Charm++ message to contain/wrap a `pupp:er`. If there's a cleaner way to do this, a more direct suggestion/example would be appreciated.

Reply:
There is this (https://charm.readthedocs.io/en/latest/charm++/manual.html#sec-messages-packed-msgs) but it doesn't look simpler than what we have now. Let's go with the current implementation, it should be fine since it's only used internally by the framework like you said.